esmvaltool/recipes/recipe_climwip_brunner2019_med.yml

```diff
@@ -0,0 +1,295 @@
+
+  - {dataset: HadGEM2-CC, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: HadGEM2-ES, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: HadGEM2-ES, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: HadGEM2-ES, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+  - {dataset: HadGEM2-ES, project: CMIP5, exp: [historical, rcp85], ensemble: r4i1p1}
+
+  - {dataset: inmcm4, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r4i1p1}
+
+  - {dataset: IPSL-CM5A-MR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: IPSL-CM5B-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MIROC-ESM-CHEM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MIROC-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  # MIROC5 has extended historical runs (overlap with RCP85) the pre-processor can't merge
+  # the first ensemble member for rsus (for the other ones it works)
+  # - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+
+  - {dataset: MPI-ESM-MR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MRI-CGCM3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MRI-ESM1, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: NorESM1-ME, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: NorESM1-M, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+    mask_landsea:
+      mask_out: sea
+
+  climatological_mean: &mean
+    climate_statistics:
+      operator: mean
+
+  climatological_std: &std
+    climate_statistics:
+      operator: std_dev
+
+  region: &region
+    <<: *general
+    extract_shape:
+      shapefile: 'shapefiles/SREX/MED/MED.shp'  # <-----
```

Comment:
I don't think this needs to be in quotes

Reply:
Do you provide the shapefiles somewhere? I could not find them. But have not worked with shapefiles before....

Reply:
@ruthlorenz : ah yes sorry - this is kind of still in progress. I think for now it will make sense to just add the 3 shapefiles needed for the paper to ClimWIP but there is a larger discussion about how to do that here: https://github.com/ESMValGroup/ESMValCore/issues/758